apps/client/src/app/components/home-watchlist/gf-create-watchlist-item-dialog/gf-create-watchlist-item-dialog.component.ts

```diff
@@ -33,16 +33,16 @@ import { Subject } from 'rxjs';
     ReactiveFormsModule
   ],
   selector: 'gf-create-watchlist-item-dialog',
-  styleUrls: ['./create-watchlist-item-dialog.component.scss'],
-  templateUrl: 'create-watchlist-item-dialog.html'
+  styleUrls: ['./gf-create-watchlist-item-dialog.component.scss'],
+  templateUrl: 'gf-create-watchlist-item-dialog.html'
 })
-export class CreateWatchlistItemDialogComponent implements OnInit, OnDestroy {
+export class GfCreateWatchlistItemDialogComponent implements OnInit, OnDestroy {
```

Comment:
While the issue description could have been clearer, the only expected change was adding this prefix.